lib/components/form/settings-selector-panel.js

```diff
@@ -137,6 +274,7 @@ class SettingsSelectorPanel extends Component {
                 <ModeButton
                   enabled
                   active={modeHasTransit && this._modeIsActive(mode)}
+                  company={mode.company}
```

Comment:
The company prop in ModeButton doesn't appear to be used. Should this be removed? (`this.props.mode.company` is used in ModeButton.)

Reply:
Good catch, will remove.